project/src/store/reducer.ts

```diff
@@ -0,0 +1,31 @@
+import { ActionType } from '../const';
+import { Actions } from '../types/action';
+import { State } from '../types/state';
+import { CITIES } from '../const';
+
+const CURRENT_CITY = CITIES.Paris.name;
+
+const initialState: State = {
+  currentCity: CURRENT_CITY ,
+  cityOffers: [],
+  offers: [],
+};
+
+const reducer = (state: State = initialState, action: Actions): State => {
+  switch (action.type) {
+    case ActionType.SetCity:
+      return {...state,
+        currentCity: action.payload,
+        cityOffers: state.offers.filter((offer) => offer.city.name === action.payload),
+      };
```

Comment:
```suggestion
    case ActionType.SetCity: {
      return {...state,
        currentCity: action.payload,
        cityOffers: state.offers.filter((offer) => offer.city.name === action.payload),
      };
      }
```
во всех кейсах должны быть курли